apps/condo/pages/employee/[id]/index.tsx

```diff
@@ -21,6 +21,34 @@ import { useRouter } from 'next/router'
 import React, { useState } from 'react'
 import { ReturnBackHeaderAction } from '@condo/domains/common/components/HeaderActions'
 
+const ReInviteActionAlert = ({ employee }) => {
+    const intl = useIntl()
+    const EmployeeDidntEnteredMessage = intl.formatMessage({ id: 'employee.EmployeeDidntEntered' })
+    const EmployeeRejectedMessage = intl.formatMessage({ id: 'pages.users.status.Rejected' })
+
+    const isEmployeeRejected = get(employee, 'isRejected')
+
+    if (isEmployeeRejected) {
+        return (
+            <Alert showIcon type='warning' message={
+                <>
+                    {EmployeeRejectedMessage}.
+                </>
+            }/>
```

Comment:
You can pass text directly in message...and use "." in lang translation, it shouldn't be hard-coded
```suggestion
            <Alert showIcon type='warning' message={EmployeeRejectedMessage}/>
```